examples/inverse/plot_rap_music.py

```diff
@@ -0,0 +1,54 @@
+"""
+=========================================
+Compute Rap-Music on evoked data
+=========================================
+
+Compute a Recursively Applied and Projected MUltiple Signal Classification
+(RAP-MUSIC) on evoked dataset.
+
+The reference for Rap-Music is:
+J.C. Mosher and R.M. Leahy. 1999. Source localization using recursively
+applied and projected (RAP) MUSIC. Trans. Sig. Proc. 47, 2
+(February 1999), 332-340.
+DOI=10.1109/78.740118 http://dx.doi.org/10.1109/78.740118
+"""
+
+# Author: Yousra Bekhti <yousra.bekhti@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+
+from mne.datasets import sample
+from mne.beamformer import rap_music
+from mne.io.pick import pick_types_evoked
+from mne.viz._3d import plot_dipoles
+
+data_path = sample.data_path()
+subjects_dir = data_path + '/subjects'
+fwd_fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
+evoked_fname = data_path + '/MEG/sample/sample_audvis-ave.fif'
+cov_fname = data_path + '/MEG/sample/sample_audvis-cov.fif'
+
+# Read the evoked response and crop it
+condition = 'Left Auditory'
+evoked = mne.read_evokeds(evoked_fname, condition=condition,
+                          baseline=(None, 0))
+evoked.crop(tmin=-50e-3, tmax=300e-3)
+
+evoked = pick_types_evoked(evoked, meg=True, eeg=False)
+
+# Read the forward solution
+forward = mne.read_forward_solution(fwd_fname, surf_ori=True,
+                                    force_fixed=False)
+
+# Read noise covariance matrix and regularize it
+noise_cov = mne.read_cov(cov_fname)
+
+dipole, residual = rap_music(evoked, forward, noise_cov, n_dipoles=2,
+                             return_residual=True, verbose=True)
+plot_dipoles(dipole, forward['src'])
```

Comment:
apparently you don't show the bem mesh anymore. Dipole fits is not meant to require a source space so I'd prefer if plot_dipoles were to take a subject and subjects_dir param to read the meshed from freesurfer folders.

so please remove the src param